lib/src/flutter_typeahead.dart

```diff
@@ -893,26 +901,45 @@ class _TypeAheadFieldState<T> extends State<TypeAheadField<T>>
         }
       }
 
-      return Positioned(
-        width: w,
-        child: CompositedTransformFollower(
-          link: this._layerLink,
-          showWhenUnlinked: false,
-          offset: Offset(
-              widget.suggestionsBoxDecoration.offsetX,
-              _suggestionsBox!.direction == AxisDirection.down
-                  ? _suggestionsBox!.textBoxHeight +
-                      widget.suggestionsBoxVerticalOffset
-                  : _suggestionsBox!.directionUpOffset),
-          child: _suggestionsBox!.direction == AxisDirection.down
-              ? suggestionsList
-              : FractionalTranslation(
-                  translation:
-                      Offset(0.0, -1.0), // visually flips list to go up
-                  child: suggestionsList,
-                ),
-        ),
+      final Widget compositedFollower = CompositedTransformFollower(
+        link: this._layerLink,
+        showWhenUnlinked: false,
+        offset: Offset(
+            widget.suggestionsBoxDecoration.offsetX,
+            _suggestionsBox!.direction == AxisDirection.down
+                ? _suggestionsBox!.textBoxHeight +
+                    widget.suggestionsBoxVerticalOffset
+                : _suggestionsBox!.directionUpOffset),
+        child: _suggestionsBox!.direction == AxisDirection.down
+            ? suggestionsList
+            : FractionalTranslation(
+                translation: Offset(0.0, -1.0), // visually flips list to go up
+                child: suggestionsList,
+              ),
       );
+
+      // When wrapped in the Positioned widget, the suggestions box widget
+      // is placed before the Scaffold semantically. In order to have the
+      // suggestions box navigable from the search input or keyboard,
+      // Semantics > Align > ConstrainedBox are needed. This does not change
+      // the style visually. However, when VO/TB are not enabled it is
+      // necessary to use the Positioned widget to allow the elements to be
+      // properly tappable.
+      return MediaQuery.of(context).accessibleNavigation
```

Comment:
This will cause appium to be unable to select item

Reply:
Why? What exactly is the issue? To a human this appears to look decent and okay ...

Reply:
I need more time to pinpoint the root cause, but for now it looks like Align is causing the problem.